assets/src/block-editor/components/with-media-library-notice.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { FeaturedImageToolbarSelect, getSelectMediaFrame } from '../../common/components/select-media-frame';
+import { setImageFromURL } from '../../common/helpers';
+import { dispatch } from '@wordpress/data';
+
+const { wp } = window;
+
+/**
+ * Gets a wrapped version of MediaUpload to display a notice for small images.
+ *
+ * Only applies to the MediaUpload in the Featured Image component, PostFeaturedImage.
+ * Mostly copied from customize-controls.js.
```

Comment:
For copied files, it makes sense to record the version that was copied, for easier checking if upstream changes need to be merged in the future.

Reply:
Great idea, https://github.com/ampproject/amp-wp/pull/3177/commits/c971ce903b7ce08cd810a763c79751338f0185e9 adds a `@link` tag.